tools/universal_file_generator_pandoc.py

```
@@ -646,11 +670,11 @@ def _add_files_to_zip(self, zf, data):
                     elif 'url' in item:
                         # Download from URL
                         try:
-                            response = requests.get(item['url'], timeout=10)
-                            if response.status_code == 200:
-                                zf.writestr(path, response.content)
+                            response = requests.get(item['url'], timeout=10, allow_redirects=_http_allow_redirects())
                         except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)